modules/collecting-gitops-debugging-data.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assembly:
+//
+// * cicd/gitops/collecting-debugging-data-for-support.adoc
+
+:_content-type: PROCEDURE
+[id="collecting-debugging-data-for-gitops_{context}"]
+= Collecting debugging data for {gitops-title}
+
+Use the `oc adm must-gather` CLI command to collect the following details about the cluster that is associated with {gitops-title}:
+
+* The subscription and namespace of the {gitops-title} Operator
+* The namespaces where ArgoCD objects are available and the objects in those namespaces, such as `ArgoCD`, `Applications`, `ApplicationSets`, `AppProjects`, and `configmaps`
+* A list of the namespaces that are managed by {gitops-title} Operator and resources from those namespaces
+* All GitOps-related custom resource objects and definitions
+* Operator and Argo CD logs
+* Warning and error-level events
+
+.Prerequisites
+* You have logged in to the {product-title} cluster as an administrator.
+* You have installed the {product-title} CLI (`oc`).
+* You have installed the {gitops-title} Operator.
+
+.Procedure
+
+. Navigate to the directory where you want to store the debugging information.
+. Run the `oc adm must-gather` command with the {gitops-title} must-gather image:
++
+[source,terminal]
+----
+$ oc adm must-gather --image=registry.redhat.io/openshift-gitops-1/gitops-must-gather-rhel8:[version-tag]
```

Comment:
@reginapizza Are we using **registry.redhat.io**?

Reply:
@reginapizza Are we targeting productization of must gather image for v1.9?

Reply:
yes, productization is in progress and will be done soon. That will be the correct tag for the must-gather, but note that the version tag will be for v1.9+ since the tag depends on the version it ships with. So for now, the only image that will work is `--image=registry.redhat.io/openshift-gitops-1/gitops-must-gather-rhel8:v1.9.0`

Reply:
@reginapizza @varshab1210 I have edited the command as per the suggestion. Please verify the PR and confirm. Thanks